srcpkgs/anydesk/template

```diff
@@ -0,0 +1,35 @@
+# Template file for 'anydesk'
+pkgname=anydesk
+version=4.0.1
+revision=1
+archs="i686 x86_64"
+wrksrc=${pkgname}-${version}
+short_desc="Fast remote desktop application"
+maintainer="mobinmob <mobinmob@disroot.org>"
+license="Free for personal use"
+#Terms and conditions:https://anydesk.com/en/terms
+homepage="https://anydesk.com/remote-desktop"
+nopie=yes
+restricted=yes
+
+if [ "${XBPS_TARGET_MACHINE}" = "x86_64" ] ; then
+	distfiles="https://download.anydesk.com/linux/anydesk-${version}-amd64.tar.gz"
+	checksum=c20b2bc265846563fb64bdb073279e651c34c43dd84b9169b5615d9b9eee6d22
+elif [ "${XBPS_TARGET_MACHINE}" = "i686" ]; then
+	distfiles="https://download.anydesk.com/linux/anydesk-${version}-i686.tar.gz"
+	checksum=029e97ff8d1be50b2d439dfc832d8d8a1e7df8e8099f96fd9b737d3204692587
+fi
+
+do_install() {
+	vmkdir usr/share/icons/anydesk
+	cp -R icons/* "${DESTDIR}"/usr/share/icons/anydesk
+	vmkdir usr/share/applications
+	vinstall anydesk.desktop 0644 usr/share/applications
+	vmkdir usr/share/polkit-1/actions
+	cp polkit-1/com.philandro.anydesk.policy "${DESTDIR}"/usr/share/polkit-1/actions/
```

Comment:
Why not use `vcopy` here?

Reply:
It is not working with vcopy, no idea why :(

Reply:
Fixed.